<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>L2 Diagnostic</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="btnStart">
        <property name="text">
         <string>Connect L2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStop">
        <property name="text">
         <string>Disconnect L2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnConfig">
        <property name="text">
         <string>Config</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnStartRotation">
        <property name="text">
         <string>Start Rotation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStopRotation">
        <property name="text">
         <string>Stop Rotation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnResetL2">
        <property name="text">
         <string>Reset L2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnVersion">
        <property name="text">
         <string>GetVersion info</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lblPacketStats">
      <property name="text">
       <string>Timestamp: Seconds     0   nsec:        0   Packet Totals:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblVersion">
      <property name="text">
       <string>HW Version: 0.0.0  FW Version: 0.0.0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblACK">
      <property name="text">
       <string>ACK results:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblIMU">
      <property name="text">
       <string>IMU:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblPCL">
      <property name="text">
       <string>PCL:</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QChartView" name="chartViewRate" native="true"/>
    </item>
    <item>
     <widget class="QLabel" name="lblRate">
      <property name="text">
       <string>Rate: 0 pkt/s</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
